spec/views/recipes_index_spec.rb

```diff
@@ -0,0 +1,26 @@
+RSpec.describe 'Recipe index', type: :feature do
+  before :each do
+    visit new_user_session_path
+    fill_in 'Email', with: 'charleslengushuru@gmail.com'
+    fill_in 'Password', with: '12345678'
+    click_button 'Log in'
+    visit recipes_path
+  end
+
+  it 'displays the Recipe page title' do
+    expect(page).to have_content('Recipes')
+  end
+
+  it 'displays the recipe name and description' do
+    Recipe.all.each do |recipe|
+      expect(page).to have_content(recipe.name)
+      expect(page).to have_content(recipe.description)
+    end
+  end
+
+  it 'displays a delete button for each recipe' do
+    Recipe.all.each do |_recipe|
+      expect(page).to have_button('Delete', count: Recipe.count)
+    end
```

Comment:
- [ ] Great job adding tests, unfortunately, three tests are failing. Kindly work on it and make sure all tests pass.

![tests](https://github.com/Odongo006/Recipe-App/assets/53124548/d7a0889c-ef4a-4f7a-917e-9f196dc06a5c)